### PR TITLE
OCPBUGS8700: correct kustomize language

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.12, 4.13+

Issue:
https://issues.redhat.com/browse/OCPBUGS-8700

Link to docs preview:
https://58375--docspreview.netlify.app/microshift/latest/microshift_running_apps/microshift-applications.html#microshift-manifests-overview_applications-microshift

QE review:
- [x] QE has approved this change.

Additional info: Bug filed on 4.12, but 4.13 docs read the same and the same corrections apply.

### CODE REVIEW

**Comment by ShaunaDiaz** [comment]
@dhellmann PTAL and LGTM if all good, or provide comments.

File: modules/microshift-manifests-overview.adoc [hunk 1]
```diff
@@ -6,12 +6,7 @@
 [id="microshift-manifests-overview_{context}"]
 = How manifests work with kustomize
 
-Providing multiple directories allows a flexible method of managing {product-title} workloads. When you run the `kustomize` configuration management tool, {product-title} searches the `/etc/microshift/manifests` and `/usr/lib/microshift/` manifest directories for a `kustomization.yaml` file. If it finds one, {product-title} automatically runs the `kubectl apply -k` command to apply the manifests to the cluster.
-
-[IMPORTANT]
-====
-The `kustomize` tool must be either running as a separate step in the boot process, or it must be part of the {product-title} image.
-====
+The `kustomize` configuration management tool is integrated with {product-title}. At every start, {product-title} searches the `/etc/microshift/manifests` and `/usr/lib/microshift/` manifest directories for a `kustomization.yaml` file. If it finds one, {product-title} automatically runs the `kubectl apply -k` command to apply the identified manifests to the cluster.
```
Comment (dhellmann) [inline modules/microshift-manifests-overview.adoc:9]:
```suggestion
The `kustomize` configuration management tool is integrated with {product-title}. At every start, {product-title} searches the `/etc/microshift/manifests` and `/usr/lib/microshift/` manifest directories for a `kustomization.yaml` file. If it finds one, {product-title} automatically runs the equivalent of the `kubectl apply -k` command to apply the identified manifests to the cluster.
```

It doesn't actually run the "kubectl" command. The same code is built into microshift, but it runs differently.

**Review by dhellmann — COMMENTED** [review]
/lgtm

**Comment by ShaunaDiaz** [comment]
@dhensel-rh Can you provide QE LGTM on this one?

**Comment by dhensel-rh** [comment]
Step C. Place the YAML file in the directory:

results in this response 

`-bash: /etc/microshift/manifests/busybox.yaml: Permission denied`

**Comment by dhensel-rh** [comment]
> @dhensel-rh Can you provide QE LGTM on this one?

@ShaunaDiaz Looking now

**Comment by dhensel-rh** [comment]
I am not sure if sudo can be used on 1C and 2A as written. I am getting permission denied. Switching to root worked for me.

**Comment by ShaunaDiaz** [comment]
@dhensel-rh Could you please double-check me now that we've updated the commands?

**Review by skrthomas — COMMENTED** [review]
/lgtm

**Pull request merged**